server_installation/topics/operator/installation.adoc

```diff
@@ -152,6 +174,7 @@ However, if you want to start tracking all Operator activities before creating c
 endif::[]
 
 * For more information on OpenShift Operators, see the link:https://docs.openshift.com/container-platform/4.4/operators/olm-what-operators-are.html[OpenShift Operators guide].
+endif::[]
```

Comment:
I saw that the build was broken due to internal anchors not working. It seems to me that this `endif::[]` should be in line 85 just before the next `ifeval::[]` starts.

Reply:
A very good catch, thank you! You are of course correct. Should be fixed now.